README.md

```diff
@@ -126,7 +126,11 @@ To build/test the project locally on your computer:
 4. __Run the tests__<br>
 `npm test`
 
+5. __Run the linter__<br>
+`npm run lint`
```

Comment:
doesn't the test comand run `lint`? 
https://github.com/APIDevTools/swagger-parser/blob/main/package.json#L43

Reply:
@P0lip 
Nice catch! Yes, seems redundant! Backing out the change.
Btw, the `npm run test` takes way longer to finish as it downloads and validates every single Swagger/OpenAPI file in apis.guru site as part of `real-world` test suite. Should we limit it to a reasonable number than running through them all?

Reply:
I feel like it's okay to validate all of them by default - likely to be difficult to distinguish the challenging ones from the easier ones.
Perhaps having an option to choose the APIs would be nice.
Thoughts?

Reply:
Hi @P0lip , sorry I was on vacation and hence is the delayed response. There are about 1500 APIs that the test suite downloads and validates. No harm in validating them all, but I like the idea of choosing which APIs to validate to expedite the run of tests. But given the 1500 number, not sure if it will be easy to configure which ones to be tested. I was thinking to limit it from 1500 to 200 or some such number and we can keep that configurable for one to change it if needed.